examples/dashboard/src/main.rs

```diff
@@ -0,0 +1,111 @@
+// To run the example with the UI:
+// cargo run --bin dashboard
+use crate::{kv::KeyValue, server::OmniPaxosServer, util::*};
+use omnipaxos::{messages::Message, util::NodeId, *};
+use omnipaxos_storage::memory_storage::MemoryStorage;
+use omnipaxos_ui::OmniPaxosUI;
+use std::{
+    collections::HashMap,
+    sync::{Arc, Mutex},
+};
+use tokio::{runtime::Builder, sync::mpsc};
+
+mod kv;
+mod server;
+mod util;
+
+type OmniPaxosKV = OmniPaxos<KeyValue, MemoryStorage<KeyValue>>;
+
+const SERVERS: [u64; 11] = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+
+#[allow(clippy::type_complexity)]
+fn initialise_channels() -> (
+    HashMap<NodeId, mpsc::Sender<Message<KeyValue>>>,
+    HashMap<NodeId, mpsc::Receiver<Message<KeyValue>>>,
+) {
+    let mut sender_channels = HashMap::new();
+    let mut receiver_channels = HashMap::new();
+
+    for pid in SERVERS {
+        let (sender, receiver) = mpsc::channel(BUFFER_SIZE);
+        sender_channels.insert(pid, sender);
+        receiver_channels.insert(pid, receiver);
+    }
+    (sender_channels, receiver_channels)
+}
+
+fn main() {
```

Comment:
add some docs (`///`) here at the top that describes what the main does.

Then also add some throughout the function so that it is easy to understand what's going on.